### PR TITLE
Homework Submission

### PR DESCRIPTION
**Pull request by dgorden**
Submitting assignments for week 3.

### CODE REVIEW

File: 01_Core/week03/submissions/exercise3/exercise3.css [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+.div1 {
+    width: 20px;
+    height: 220px;
+    background-color: white;
+    margin: 0 auto;
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    z-index: 1;
```
Comment (broffis) [inline 01_Core/week03/submissions/exercise3/exercise3.css:21]: Be careful using z-index, especially if the intent is to keep everything on the same plane. You'll quickly find that things jump around and don't line up properly

File: 01_Core/week03/submissions/exercise3/exercise3.css [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    height: 220px;
+    margin: 0 auto;
+    display: grid;
+    background-color: blue;
+    z-index: 0;
+    position: relative;
+    border: 10px solid white;
+}
+
+.div1 {
```
Comment (broffis) [inline 01_Core/week03/submissions/exercise3/exercise3.css:12]: You repeat a good bit of this styling across a few of the divs. Try to keep shared styling within shared classes/tags

File: 01_Core/week03/submissions/exercise3/exercise3.css [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+
+  .div5 {
+    /* margin: 50px 0; */
+    position: absolute;
+    display: block;
+    color: black;
+    width: 0px;
+    height: 0px;
+    border-right: 13px solid transparent;
```
Comment (broffis) [inline 01_Core/week03/submissions/exercise3/exercise3.css:69]: Is this how you got the star in there?

File: 01_Core/week03/submissions/exercise3/exercise3.css (hunk 1 above)
Comment (dgorden) [reply 01_Core/week03/submissions/exercise3/exercise3.css:21]: Will be cognizant of that going forward. In this situation, I used Z-Index to position elements in front of other elements. Use case for that was to get the star in front of the circle, the circle in front of the "cross" divs, and the "cross" divs in front of the blue square div. 

I'd be happy to learn about alternative solutions if you have any you'd recommend! 

Thank you for the feedback!

File: 01_Core/week03/submissions/exercise3/exercise3.css (hunk 2 above)
Comment (dgorden) [reply 01_Core/week03/submissions/exercise3/exercise3.css:12]: This is a great point, and I was hoping to refactor some of the wet code before submitting. Will try to get ahead of that in the future!

File: 01_Core/week03/submissions/exercise3/exercise3.css (hunk 3 above)
Comment (dgorden) [reply 01_Core/week03/submissions/exercise3/exercise3.css:69]: The star starts at line 62 through the rest of the stylesheet. It uses psuedo elements at a width/height of 0, with borders going around to create three triangles that are positioned at various degrees to create a start shape.

File: 01_Core/week03/submissions/exercise7/exercise7.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+.interior {
+    /* display: grid; */
+    width: 400px;
+    height: 100px;
+    border-top: 12.5px solid white;
+    border-left: 12.5px solid white;
+}
+
+.div1 {
+    background: linear-gradient(90deg, white 0%, white 5%, rgba(255,0,0,.5) 5%, rgba(255,0,0,.5) 30%, white 30%, white 35%, rgba(255,0,0,.75) 35%, rgba(255,0,0,.75) 60%, white 60%,
```
Comment (broffis) [inline 01_Core/week03/submissions/exercise7/exercise7.css:19]: This is a really impressive way to create the gradient. Not something I would have thought to do. If you want an extra challenge, try to recreate the image using opacity and `:before` `:after`

Comment (dgorden) [reply 01_Core/week03/submissions/exercise7/exercise7.css:19]: Thanks for the feedback! I'll have to circle back and experiment with :before/:after!

File: 01_Core/week03/submissions/exercise3/exercise3.css (hunk 3 above)
Comment (broffis) [reply 01_Core/week03/submissions/exercise3/exercise3.css:69]: 1) That's impressive. Not something I would have thought of
2) You'll probably hate me for this, but that's grossly overcomplicated. You can actually use the `content: ''` style you have to add the image that was included

**Pull request closed without merging**